dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -171,6 +175,58 @@ const buttonExpanded = css`
 	}
 `;
 
+const badgeDiameter = 20;
+const notificationColor = palette.error[400];
+
+const notificationBadgeStyles = css`
+	background-color: ${notificationColor};
+	color: ${palette.neutral[100]};
+	position: absolute;
+	top: 0;
+	left: 0;
+	min-width: ${badgeDiameter}px;
+	height: ${badgeDiameter}px;
+	border-radius: ${badgeDiameter / 2}px;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 0 5px;
+	margin-left: -10px;
+	margin-top: -3px;
+`;
+
+const notificationStyles = css`
+	${textSans.xxsmall()};
+	color: ${notificationColor};
+`;
+
+const redDotDiameter = 16;
+const redDotStyles = css`
+	background-color: ${notificationColor};
+	width: ${redDotDiameter}px;
+	height: ${redDotDiameter}px;
+	border-radius: ${redDotDiameter / 2}px;
+	position: absolute;
+	left: 5px;
+	top: 12px;
+`;
+
+const notificationCountStyles = css`
+	${textSans.xsmall()};
+	line-height: ${badgeDiameter}px;
+`;
+
+const NotificationBadge = ({ count }: { count: number }) => {
+	return (
+		<div css={notificationBadgeStyles}>
+			<span css={notificationCountStyles}>{count + 0}</span>
```

Comment:
Why the `count + 0` here? Were there odd errors?

Reply:
Hmm this must have been left over from something I tried during development. It's definitely not adding anything, I'll remove.